Prefer kchmviewer over Okular, since Okular is atrocious at displaying reflowable formats.
Note that Okular's InitialPreference is 3 for chm, 1 for epub.

--- a/packages/freedesktop/uchmviewer.desktop	2021-05-20 09:09:01.000000000 +0200
+++ b/packages/freedesktop/uchmviewer.desktop	2022-02-17 12:09:34.471436243 +0100
@@ -7,4 +7,5 @@
 Icon=uchmviewer
 GenericName=CHM file viewer
 Comment=A viewer of CHM (MS-Help) files
+InitialPreference=8
 MimeType=application/chm;application/x-chm;application/vnd.ms-htmlhelp;application/epub+zip
